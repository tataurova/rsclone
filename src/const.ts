export const RouteName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};
export const PageName = {
  MAIN: '/',
  LOGIN: '/login',
  GONE: '/gone',
  LOOKING_RELATIVES: '/looking-relatives',
  ACTIVE_SEARCHES: '/active-searches',
  CLOSED_SEARCHES: '/closed-searches',
};

export const AuthorizationStatus = {
  AUTH: 'AUTH',
  NO_AUTH: 'NO_AUTH',
};

export const Error = {
  UNAUTHORIZED: 401,
  BAD_REQUEST: 400,
  NOT_FOUND: 404,
  SERVICE_UNAVAILABLE: 503,
  TIMEOUT_ERROR: 'timeout_error',
};

export const headCells = [
  { id: 'status', label: 'Статус' },
  { id: 'city', label: 'Город' },
  { id: 'name', label: 'ФИО' },
  { id: 'age', label: 'Возраст' },
  { id: 'date', label: 'Дата' },
  { id: 'coordinator', label: 'Координатор' },
  { id: 'people', label: 'Людей на поиске' },
  { id: 'actions', label: 'Действия', disableSorting: true },
];

export const getStatusCollection = [
  'Резерв',
  'Поиск на месте',
  'Стоп. Проверка информации',
  'Архив',
];

export const SearcherStatus = {
  way: 'В пути',
  place: 'На месте',
  home: 'Дома',
};

export const TransportStatus = {
  car: 'На машине',
  own: 'Своим ходом',
  need: 'Нужна машина',
};

export const MAX_TIMEOUT = 5000;
export const SHOW_ERROR_TIMEOUT = 3000;
