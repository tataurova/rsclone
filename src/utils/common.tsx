export const extend = (a, b) => ({ ...a, ...b });
