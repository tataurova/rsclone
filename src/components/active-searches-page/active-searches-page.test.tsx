import * as React from 'react';
import * as renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import ActiveSearchesPage from './active-searches-page';
import { mockActiveSearches, mockClosedSearches } from '../../mock';
import { AuthorizationStatus } from '../../const';
import { mockFunction } from '../../utils/common';

describe('<ActiveSearchesPage />', () => {
  const mockStore = configureStore([]);
  const store = mockStore({
    DATA: {
      gonePeople: [],
      lookingRelativesPeople: [],
      activeSearches: mockActiveSearches,
      closedSearches: mockClosedSearches,
      isFetching: false,
      error: false,
    },
    APP: {
      page: '3',
    },
    AUTH: {
      authorizationStatus: AuthorizationStatus.NO_AUTH,
      user: '',
    },
  });

  it('Render ActiveSearchesPage', () => {
    const tree = renderer
      .create(
                <Provider store={store}>
                    <ActiveSearchesPage
                        page={'3'}
                        activeSearches={mockActiveSearches}
                        closedSearches={mockActiveSearches}
                        onAddActiveSearches={mockFunction}
                        onAddClosedSearches={mockFunction}
                    />
                </Provider>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
