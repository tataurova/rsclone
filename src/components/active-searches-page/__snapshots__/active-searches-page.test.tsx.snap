// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActiveSearchesPage /> Render ActiveSearchesPage 1`] = `
Array [
  <div
    className="makeStyles-root-6"
  >
    <header
      className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <button
          aria-label="menu"
          className="MuiButtonBase-root MuiIconButton-root makeStyles-menuButton-7 MuiIconButton-colorInherit MuiIconButton-edgeStart"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiIconButton-label"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              onClick={[Function]}
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
          </span>
        </button>
        <h6
          className="MuiTypography-root makeStyles-title-8 MuiTypography-h6"
        >
          Active searches
        </h6>
        <img
          alt="Logo"
          className="makeStyles-img-9"
          height="50px"
          src=""
          width="50px"
        />
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Login
          </span>
        </button>
      </div>
    </header>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-10 makeStyles-root-1"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular makeStyles-height-4 MuiToolbar-gutters"
    >
      <div
        className="MuiFormControl-root MuiTextField-root"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined"
          data-shrink={true}
        >
          Search
        </label>
        <div
          className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart"
          onClick={[Function]}
        >
          <div
            className="MuiInputAdornment-root MuiInputAdornment-positionStart"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
              />
            </svg>
          </div>
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart"
            disabled={false}
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
          <fieldset
            aria-hidden={true}
            className="PrivateNotchedOutline-root-11 MuiOutlinedInput-notchedOutline"
          >
            <legend
              className="PrivateNotchedOutline-legendLabelled-13 PrivateNotchedOutline-legendNotched-14"
            >
              <span>
                Search
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <button
        className="MuiButtonBase-root MuiButton-root makeStyles-root-15 MuiButton-outlined makeStyles-newButton-3 MuiButton-outlinedPrimary MuiButton-outlinedSizeLarge MuiButton-sizeLarge"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label makeStyles-label-16"
        >
          <span
            className="MuiButton-startIcon MuiButton-iconSizeLarge"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
          </span>
          Add
        </span>
      </button>
    </div>
    <table
      className="MuiTable-root makeStyles-table-5"
      role={null}
    >
      <thead
        className="MuiTableHead-root"
        role={null}
      >
        <tr
          className="MuiTableRow-root MuiTableRow-head"
          role={null}
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              Status
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              City
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              Name
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              Age
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              Date
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              Coordinator
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            <span
              aria-disabled={false}
              className="MuiButtonBase-root MuiTableSortLabel-root"
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={0}
            >
              People on the search
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                />
              </svg>
            </span>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            Actions
          </th>
        </tr>
      </thead>
      <tbody
        className="MuiTableBody-root"
        role={null}
      >
        <tr
          className="MuiTableRow-root"
          role={null}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Поиск на месте
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Санкт-Петербург
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Петр
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            30
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            01-12-2020
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Чиж
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            8
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-17 makeStyles-primary-19"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
              </span>
            </button>
          </td>
        </tr>
        <tr
          className="MuiTableRow-root"
          role={null}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Стоп. Проверка информации
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Минск
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Аркадий
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            100
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            01-01-2021
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Чип
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            4
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-17 makeStyles-primary-19"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
              </span>
            </button>
          </td>
        </tr>
        <tr
          className="MuiTableRow-root"
          role={null}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Поиск на месте
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Москва
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Владимир
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            3000
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            06-12-2020
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Дейл
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            4
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-17 makeStyles-primary-19"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
              </span>
            </button>
          </td>
        </tr>
        <tr
          className="MuiTableRow-root"
          role={null}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Резерв
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Санкт-Петербург
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Петр
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            30
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            01-12-2020
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Чи
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          />
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-17 makeStyles-primary-19"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
              </span>
            </button>
          </td>
        </tr>
        <tr
          className="MuiTableRow-root"
          role={null}
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Поиск на месте
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Киев
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Анна
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            123
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            01-12-2020
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            Кат
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            4
          </td>
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body"
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-17 makeStyles-primary-19"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M14.06 9.02l.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
              </span>
            </button>
          </td>
        </tr>
      </tbody>
    </table>
    <div
      className="MuiTablePagination-root"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular MuiTablePagination-toolbar MuiToolbar-gutters"
      >
        <div
          className="MuiTablePagination-spacer"
        />
        <p
          className="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
        >
          Rows per page
        </p>
        <div
          className="MuiInputBase-root MuiTablePagination-input MuiTablePagination-selectRoot"
          onClick={[Function]}
        >
          <div
            aria-haspopup="listbox"
            className="MuiSelect-root MuiSelect-select MuiTablePagination-select MuiSelect-selectMenu MuiInputBase-input"
            onBlur={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onMouseDown={[Function]}
            role="button"
            tabIndex={0}
          >
            5
          </div>
          <input
            aria-hidden={true}
            className="MuiSelect-nativeInput"
            onAnimationStart={[Function]}
            onChange={[Function]}
            tabIndex={-1}
            value={5}
          />
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSelect-icon MuiTablePagination-selectIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
        </div>
        <p
          className="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
        >
          1-5 from 5
        </p>
        <div
          className="MuiTablePagination-actions"
        >
          <button
            aria-label="Previous page"
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
            disabled={true}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={-1}
            title="Previous page"
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                />
              </svg>
            </span>
          </button>
          <button
            aria-label="Next page"
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
            disabled={true}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={-1}
            title="Next page"
            type="button"
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
  <footer
    className="makeStyles-footer-24"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthLg"
    >
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
      >
        © 
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkFont-26 MuiTypography-colorInherit"
          href="https://rs.school/js/"
          onBlur={[Function]}
          onFocus={[Function]}
          target="_blank"
        >
          RS School
        </a>
         
        <span
          className="makeStyles-flex-25"
        >
          <span
            className="makeStyles-link-27"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-28"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-30 MuiTypography-colorPrimary"
              href="https://github.com/tataurova/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Olga Tataurova
            </a>
          </span>
          <span
            className="makeStyles-link-27"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-28"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-30 MuiTypography-colorPrimary"
              href="https://github.com/shuliakleonid/"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Leonid Shuliak
            </a>
          </span>
          <span
            className="makeStyles-link-27"
          >
            <img
              alt="GitHub"
              className="makeStyles-img-28"
              src=""
            />
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-30 MuiTypography-colorPrimary"
              href="https://github.com/andreizayaz"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              rel="noreferrer"
              target="_blank"
            >
              Andrei Podyslenkau
            </a>
          </span>
        </span>
        2021
      </p>
    </div>
  </footer>,
]
`;
